lib/exq/redis/job_stat.ex

```diff
@@ -42,9 +42,20 @@ defmodule Exq.Redis.JobStat do
     {:ok, count}
   end
 
-  def add_process_commands(namespace, process_info, serialized_process \\ nil) do
-    serialized = serialized_process || Exq.Support.Process.encode(process_info)
-    [["SADD", JobQueue.full_key(namespace, "processes"), serialized]]
+  def add_process_commands(namespace, process_info, _) do
+    name = supervisor_worker_name(namespace, process_info)
+    string_pid = :erlang.pid_to_list(process_info.pid)
+    [
+      ["SADD", JobQueue.full_key(namespace, "processes"), name], # ensure supervisor worker is added to list
+      ["HINCRBY", name, "busy", "1"],
+      ["HSET", "#{name}:workers", string_pid, Poison.encode!(%{
```

Comment:
could we use Exq.Serializers.JsonSerializer, as we only have optional dependency on poison